packages/front-end/components/GetStarted/index.tsx

```diff
@@ -161,7 +167,19 @@ const GetStartedAndHomePage = (): React.ReactElement => {
   // Advanced Features Cards Section
 
   const advancedFeatures: AdvancedFeature[] = useMemo(() => {
-    return advancedFeatureList.sort(() => Math.random() - 0.5).slice(0, 3);
+    const advancedFeatureListWithAnalytics = [...advancedFeatureList];
+    if (isCloud()) {
+      advancedFeatureList.push({
+        imgUrl: "/images/get-started/advanced/feature-flag.png",
+        title: "Feature Flag Analytics",
+        description: "View flag evaluations in real time",
+        docSection: "managedWarehouseTracking",
+        commercialFeature: "managed-warehouse",
+      });
```

Comment:
I don't think this works?  You are pushing onto the `advancedFeatureList` array and not the `advancedFeatureListWithAnalytics` array